tools/mktutorial/main.go

```diff
@@ -369,9 +375,12 @@ func emitCloudIndexShortcode(root, cloud string, tutorials []tutorial) error {
 
 	// Render the template using the tutorials data.
 	if err = cloudIndexTemplate.FRender(f, map[string]interface{}{
```

Comment:
Is there any reason this map would be different from the previous map? In which case, can we just declare this once?

Reply:
I left things the way they already were.  Happy to do some additional refactoring.

Reply:
Actually - going to leave this as-is.  These are parameters to the templates, which could well intentionally want to diverge in the future.  No particularly great need to DRY this.